### ISSUE TITLE
Silent fail when files are too large (when server configured to limit file size to 10 MB)

### ISSUE THREAD

**Issue by SamMousa** [owncloud/client#4774]
### Expected behaviour

If a file fails to upload with `413 Request Entity Too Large` I expect the client to let me know.
### Actual behaviour

The client skips the file and reports a success status.
### Steps to reproduce
1. Install OwnCloud
2. Configure webserver and PHP to limit file size to 10MB.
3. Put a 15MB file in the folder that the sync client is using.
### Server configuration

Operating system: Ubuntu Linux 

Web server: nginx 1.9

Database: mysql

PHP version: 7

ownCloud version: 9.1.0-pre-alpha

Storage backend: local
### Client configuration

Client version: 2. 1.1

Operating system: Xubuntu and Mac OS El Capitan

OS language: English and Dutch

Installation path of client: /usr/share/owncloud-client and unknown.
### Logs

Logs on server explicitly mention http code `413`. Same for the client.

**Comment by guruz** [comment]
@SamMousa Are you sure that file is not mention in the "Not synced" tab?

**Comment by SamMousa** [comment]
I'm sure they are mentioned there.

The question is why the status indicator says "succes"; why would I check such an "error log" or in this case list of unsynced files if the summary tells me everything was synced?

**Comment by guruz** [comment]
@SamMousa What's the line in php.ini you have used? :)

What exactly do you mean by status indicator? The tray icon?

**Comment by SamMousa** [comment]
Yes the tray icon and when I open the app on the "main" screen it shows that syncing is done.

@guruz What php.ini line do you mean? The line to get rid of the error?
I'm not sure, but it is not relevant to this bug. I realize the source of the "not syncing large files" issue is configuration.
This issue however is not about not uploading files, but not correctly (in my opinion) handling a 413 entry.

**Comment by hodyroff** [comment]
Not a bug at this time, works as designed. When we redo error messages in the future we should carefully consider the different use cases. We do expect that the end user has some knowledge about his server limitations. And the not synced tab has enough information about the issue. When we have not synced items we show green. We might need to rethink how the catergories are built and will collect further user experience. Thanks for your input in any case!
@pmaier1
